index.bs

```diff
@@ -8923,11 +8923,7 @@ corresponding to [=interface members=].
 
     <small class="non-normative">
         The [{{LegacyArrayClass}}] [=extended attribute=] appears on
-        the following [=interfaces=]:
-        {{DOMRectList}},
-        {{MediaList}},
-        {{StyleSheetList}}, and
-        {{CSSRuleList}}. [[GEOMETRY]] [[CSSOM]]
+        {{DOMRectList}}. [[GEOMETRY]]
```

Comment:
maybe: "appears on the {{DOMRectList}} [=interface=]. [[GEOMETRY]]"?

Reply:
Agree it's nicer, fixed :)